gemdaqmonitor/src/common/DaqMonitor.cc

```diff
@@ -253,6 +196,7 @@ void gem::daqmon::DaqMonitor::updateDAQOHmain()
 {
   DEBUG("DaqMonitor: Update DAQ OH main table");
   req = wisc::RPCMsg("amc.getmonDAQOHmain");
+  req.set_word("NOH",NOH);
```

Comment:
I think @jsturdy raised this point, rather than setting an NOH word in the RPC msg, shouldn't the RPC module just get NOH from the FW?  This is what I've asked Andrew to do in [ctp7_modules #51](https://github.com/cms-gem-daq-project/ctp7_modules/pull/51#pullrequestreview-161229087)

Reply:
this might be updated later. Current version of RPC module is like that, so the `NOH` parameter is set. An update to daq module is planned anyway (first, this functionality will be in a different module, second, some of the functions will be altered)